ArchiSteamFarm/Commands.cs

```diff
@@ -0,0 +1,267 @@
+﻿using ArchiSteamFarm.Localization;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ArchiSteamFarm {
+	internal sealed class Commands {
+		//Because this dictionary maps to async functions we have to make sync functions async with lambdas; e.g. VERSION
+		//Because for some commands there exist versions with AND without parameters we have to use lambdas to adapt function-interfaces sometimes; e.g. STOP
+		private static readonly Dictionary<(string Command, byte ArgumentCount), Func<Bot, ulong, string[], Task<string>>> CommandDictionary = new Dictionary<(string Command, byte arguments), Func<Bot, ulong, string[], Task<string>>>() {
```

Comment:
This is wrong on so many levels. Why do you even put args number there, didn't I just tell you to make static version for each command and in that static version do any kind of arguments parsing? Why do you put lambdas here instead of JUST a pointer to a function that you will await/execute in `Parse()` function?